tests/functional_tests/ffront_tests/test_unrollPowOp.py

```diff
@@ -0,0 +1,160 @@
+import pathlib
+
+from eve.pattern_matching import ObjectPattern
+from functional.ffront import common_types as ct, field_operator_ast as foast
+from functional.ffront.func_to_foast import UnrollPowerOp
+
+
+def test_power_0():
+    def func():
+      return a**0
+    testee = FieldOperatorParser.apply_to_function(func)
+
+    actual = UnrollPowerOp().visit(testee)
+
+    expected = ObjectPattern(
+        foast.BinOp,
+        right=foast.Name(
+            id="a",
+            type=ct.DeferredSymbolType(constraint=None),
+            location=foast.SourceLocation(
+                line=106, column=12, source=str(pathlib.Path(__file__).resolve())
+            ),
```

Comment:
```suggestion
```
Specifying the location is superfluous entirely. That is the point of `ObjectPattern`. If an attribute is omitted in the pattern it will continue to match. Equally for the remaining tests.